whiteboard/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "Whiteboard",
-	"version": "1.3.0",
+	"version": "1.3.1",
```

Comment:
conflict with #45 , gotta change whichever one gets merged second to 1.3.2

Reply:
Oh true, need to fix the whiteboard.zip move too. I'll update this diff after I merge yours